test/standalone/empty_env/build.zig

```diff
@@ -7,8 +7,14 @@ pub fn build(b: *std.Build) void {
 
     const optimize: std.builtin.OptimizeMode = .Debug;
 
-    if (builtin.os.tag == .windows and builtin.cpu.arch == .aarch64) {
+    if (builtin.os.tag == .windows and
         // https://github.com/ziglang/zig/issues/13685
+        (builtin.cpu.arch == .aarch64 or
+        // ConEmu injects environment variables into processes before they are executed
+        // depending on user settings. This obviously invalidates the test, so skipping
+        // it is the best option.
+        std.process.hasEnvVarConstant("ConEmuHWND")))
```

Comment:
Please untangle these conditions so that one can be deleted cleanly when it is resolved.